### ISSUE TITLE
Add aria-label to sidenav on Benefits Detail pages

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14870]
## Describe the defect
The sidenav currently doesn't have a unique label so it cannot be differentiated clearly from other navigation landmarks on the page.
![image](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/106678594/774354d8-2884-4968-8156-12c95b49e3b7)

## To Reproduce
1. Go to a Benefits Detail page like [About VA Health Benefits](https://www.va.gov/health-care/about-va-health-benefits/)
2. Inspect the left sidenav menu
3. Notice that it is a `<nav>` element but it does not have an `aria-label` attribute

## AC / Expected behavior

- [ ] The sidenav menu should have `aria-label="Secondary"` to distinguish it from other navigation menus on the same page

## Screenshots
![image](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/106678594/774354d8-2884-4968-8156-12c95b49e3b7)

 ## Additional context
I have the [same ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1999) in for the DST but [after speaking with Randi](https://dsva.slack.com/archives/C52CL1PKQ/p1692288683071189), she said that since this is not a typical web component, updates made by the DST in this case would not impact PW templates and therefore you would need your own ticket.

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by laflannery** [comment]
@chriskim2311 I now realize there is no AC for my review but I would like to if possible? Is there a PR?

**Comment by chriskim2311** [comment]
@laflannery Yes! Here is the PR https://github.com/department-of-veterans-affairs/content-build/pull/1709

**Comment by chriskim2311** [comment]
Change is live in production, closing! @laflannery 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcWloIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--465e932ae71b449cc0e414a3ae457a116ce776e7/Cursor_and_About_VA_Health_Benefits___Veterans_Affairs.png)
